ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -33,6 +33,19 @@ object Main {
     val StartupError = 101
   }
 
+  def adjustAndReloadLoggingOptions(
+      config: Config
+  ): Unit = {
+    // If the system property was explicitly set before application startup and the config option was provided,
+    // the prior value will be overridden here.
+    if (config.useJsonLogEncoder) {
+      Logging.setUseJsonLogEncoderSystemProp()
+      Logging.reconfigure(getClass, "logback.xml")
+    }
+    // Not related to the content of the logback.xml file.
```

Comment:
not sure what this comment means

Reply:
I edited the comment, I hope it is now more clear :thinking: